### DISCUSSION TITLE
[Question][Hw2] Cannot connect to VM

### DISCUSSION THREAD

**Discussion by Byunk** [prosyslab-classroom/cs348-information-security#690] [Q&A]
Name: Kyungho Byoun

I cannot connect to VM right now after executing long loop test case.

<img width="739" alt="image" src="https://user-images.githubusercontent.com/60650372/230027226-9adf141b-356b-490e-a8c2-b7366eab2665.png">

I cannot connect via ssh as well. 

![image](https://user-images.githubusercontent.com/60650372/230027808-309d7076-34c4-4b3b-8eef-6a1abc12aa4a.png)

After type the password, it freeze right after.

**Comment by KAIST-JongchanPark** [comment]
It seems working well now.

**Comment by Byunk** [comment]
Yes. Thank you!
